packages/cli/src/util/config.ts

```diff
@@ -91,10 +96,17 @@ export async function loadFoundryConfig(): Promise<
   const configFilePath = await findUp(CONFIG_FILE_NAMES);
 
   if (configFilePath) {
+    const extension = extname(configFilePath);
+    if (!SUPPORTED_EXTENSIONS.includes(extension)) {
```

Comment:
Hmm, I'm not really sure this is useful given that the extension is fixed in the config file name. I think we'd instead in the future when adding extra supported config file names run a switch over the extension and have different parsers or a map from extension to parser rather than an array of supported extensions. The unmatched case would then be "unsupported" rather than a check like this that we have to keep the array in sync with the actual parsing branches.

Reply:
Yeah that makes sense. I updated the implementation to use switch and run it of the file extension directly. I think its much better now